### PR TITLE
Guarding against null keyring (Hardware Wallet Selectors)

### PR DESCRIPTION
**Pull request by ryanml**
Related Sentry Error: https://sentry.io/organizations/metamask/issues/2588041346

It is possible for the `getCurrentKeyring` selector to return `null`: https://github.com/MetaMask/metamask-extension/blob/develop/ui/selectors/selectors.js#L69 - we should factor in that scenario when handling results from it

### CODE REVIEW

File: ui/selectors/selectors.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ export function isHardwareWallet(state) {
  */
 export function getHardwareWalletType(state) {
   const keyring = getCurrentKeyring(state);
-  return keyring.type.includes('Hardware') ? keyring.type : undefined;
+  return keyring?.type?.includes('Hardware') ? keyring.type : undefined;
```
Comment (adonesky1) [inline ui/selectors/selectors.js:113]: [nit] this could be `isHardwareWallet(state) ? keyring.type : undefined`

Comment (ryanml) [reply ui/selectors/selectors.js]: @adonesky1 definitely could, fixed!

**Review by adonesky1 — APPROVED** [review]
LGTM!

**Review by NiranjanaBinoy — APPROVED** [review]
lgtm

**Pull request merged**